### ISSUE TITLE
mupdf version check

### ISSUE THREAD

**Issue by swt2c** [pymupdf/PyMuPDF#353]
When building on Fedora, I ran into this when using mupdf 1.16.1:

```
+ /usr/bin/python3 -c 'import sys; sys.path.remove(""); import fitz'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/builddir/build/BUILDROOT/python-PyMuPDF-1.16.0-1.fc32.x86_64/usr/lib64/python3.8/site-packages/fitz/__init__.py", line 18, in <module>
    raise ValueError(
ValueError: MuPDF library mismatch 1.16.0 <> 1.16.1
```
The MuPDF minor releases should be compatible as far as I know, so perhaps the version check could be a little looser?  Looks like the only change in 1.16.1 is an Android build fix.

**Comment by JorjMcKie** [comment]
Yes I agree - already working on a v1.16.1, where a different patch level is accepted with a warning.

**Comment by JorjMcKie** [comment]
should be fixed in v1.16.1 now.
